### ISSUE TITLE
Search returns incomplete results

### ISSUE THREAD

**Issue by LightIsLife** [openhab/website#140]
Just noticed that searching the docs for "Profile" does not deliver the link where this new concept is explained. Using the search function, I was expecting to find this link: https://www.openhab.org/docs/configuration/items.html#profiles 

However, this link was not returned by the search, see screenshot below:
<img width="597" alt="screen shot 2018-12-31 at 10 10 02 am" src="https://user-images.githubusercontent.com/24855935/50557312-63536f80-0ce4-11e9-8808-5782b731c3b0.png">

**Comment by ghys** [comment]
Indeed. My guess is it's simply not prominent enough yet to have a good ranking in the index. I don't know nor control how Algolia works in this matter. Maybe @Shipow or @s-pace can provide more insight.

But I can confirm this paragrapg is indeed indexed because searching for "built-in profiles" or "profiles specified" or "rawbutton-toggle-switch" returns results.
I don't have a good miracle suggestion, maybe the title should be changed to "Defining Profiles" or something similar.

**Comment by s-pace** [comment]
👋 @LightIsLife,
@ghys has well described the situation here. The level of the word you are looking for is wrapped in a  `<h4/>` tag, that is to say a `lvl5` in your DocSearch hierarchy [defined by your configuration](https://github.com/algolia/docsearch-configs/blob/master/configs/openhab.json). While the current most popular result here is wrapped in a `<h2/>` tag, that is to say a `lvl2` in the DocSearch hierarchy. This is why it is ranked higher.

The expected record is indexed but ignored missing because, by default, we [only display the top 5 best records](https://community.algolia.com/docsearch/behavior.html#algoliaoptions). 

You can change the HTML tag wrapping the expected record of your website or you can also change the `hitsPerPage` parameter from the DocSearch JS snippet included in your page.

**Comment by Confectrician** [comment]
Thanks for this well explanation. :)

**Comment by Confectrician** [comment]
Hey @LightIsLife,

Of course this is a valuable point.
We introduced algolia a short time ago and so it is possible that we have to adapt the settings.
I think this will not be the last time, we have to edit something.

I think 10 results should be a good compromise.
More content and still a dropdown that is not too big.

**Comment by ghys** [comment]
As long as it's still ok on phones and so on.
